homeassistant/components/abode/__init__.py

```diff
@@ -119,7 +120,7 @@ def __init__(self, abode, polling):
 
     except (AbodeException, ConnectTimeout, HTTPError) as ex:
         LOGGER.error("Unable to connect to Abode: %s", str(ex))
-        return False
+        raise PlatformNotReady
```

Comment:
There is a different exception for config entries

```suggestion
        raise ConfigEntryNotReady
```